System/SwatEquipment.ini

```diff
@@ -3918,7 +3918,7 @@ PlayerUsable=true
 EnemyUsesAmmo=(AmmoClass="SwatAmmo.M870SGAmmo",Chance=60)
 EnemyUsesAmmo=(AmmoClass="SwatAmmo.M870SG000Ammo",Chance=30) 
 EnemyUsesAmmo=(AmmoClass="SwatAmmo.M870SGSabotAmmo",Chance=10)
-MagazineSize=5
```

Comment:
There are two options. This value is also realistic

Reply:
Law enforcement would  prefer 6+1 when it comes to combat shotguns